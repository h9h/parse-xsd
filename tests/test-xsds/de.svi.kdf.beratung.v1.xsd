<?xml version="1.0" encoding="UTF-8"?>
<!--
____________________________________________________________________________________

Allgemeine Attribute in der Beratung

____________________________________________________________________________________

Revision History:
____________________________________________________________________________________

1.0 18.10.2017 - Heinke - initial
____________________________________________________________________________________
-->
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:kcom="http://svi.de/kdf/ei/common/v1" xmlns:kbrtg="http://svi.de/kdf/beratung/v1" targetNamespace="http://svi.de/kdf/beratung/v1" elementFormDefault="qualified" attributeFormDefault="unqualified">
    <xs:annotation>
        <xs:documentation>Allgemeine Attribute in der Beratung</xs:documentation>
    </xs:annotation>
    <xs:import namespace="http://svi.de/kdf/ei/common/v1" schemaLocation="de.svi.kdf.ei.common.xsd"/>
    <xs:element name="angesprocheneChance" type="kbrtg:angesprocheneChanceType">
        <xs:annotation>
            <xs:documentation>Im Falle eines Kontaktes mit CrossUpSelling-Chance die angesprochene Chance</xs:documentation>
        </xs:annotation>
    </xs:element>

    <xs:simpleType name="angesprocheneChanceType">
        <xs:annotation>
            <xs:documentation>Im Falle eines Kontaktes mit CrossUpSelling-Chance die angesprochene Chance</xs:documentation>
        </xs:annotation>
        <xs:restriction base="kcom:string200Type"/>
    </xs:simpleType>

</xs:schema>