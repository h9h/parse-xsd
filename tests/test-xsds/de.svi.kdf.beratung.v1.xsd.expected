{
  "element": {
    "annotation": {
      "documentation": "Im Falle eines Kontaktes mit CrossUpSelling-Chance die angesprochene Chance"
    },
    "name": "angesprocheneChance",
    "type": "kbrtg:angesprocheneChanceType"
  },
  "simpleType": {
    "annotation": {
      "documentation": "Im Falle eines Kontaktes mit CrossUpSelling-Chance die angesprochene Chance"
    },
    "restriction": {
      "base": "kcom:string200Type"
    },
    "name": "angesprocheneChanceType"
  },
  "meta": {
    "annotation": {
      "documentation": "Allgemeine Attribute in der Beratung"
    },
    "import": {
      "namespace": "http://svi.de/kdf/ei/common/v1",
      "schemaLocation": "de.svi.kdf.ei.common.xsd"
    },
    "xmlns:xs": "http://www.w3.org/2001/XMLSchema",
    "xmlns:kcom": "http://svi.de/kdf/ei/common/v1",
    "xmlns:kbrtg": "http://svi.de/kdf/beratung/v1",
    "targetNamespace": "http://svi.de/kdf/beratung/v1",
    "elementFormDefault": "qualified",
    "attributeFormDefault": "unqualified"
  }
}